lib/web/apiserver.go

```diff
@@ -2168,11 +2168,26 @@ func (h *Handler) deleteWebSession(w http.ResponseWriter, r *http.Request, _ htt
 		}
 	}
 
+	clt, err := ctx.GetClient()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	user, err := clt.GetUser(r.Context(), ctx.GetUser(), false)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
```

Comment:
If any of these calls fail, we won't log out the user. 
I see that in the logout handler (line 2181) we rather try to log the errors instead of returning early.